infrastructure/README.md

```diff
@@ -1,3 +1,19 @@
 # Infrastructure
+Generating kubernetes yaml file using docker-compose.yaml to launch and deploy the project inside the OC4
+- Using Kompose kubernetes tool
+ - Install Kompose and add your directory location into your environment  settings PATH variable.
+ - Inside the terminal go to your docker-compose.yaml location of the project
+ - Run command "kompose convert --provider=openshift --build=build-config --build-repo=https://github.com/bcgov/jag-traffic-courts-online.git" this will convert all the service inside the docker-compose.yaml to kubernetes yaml format and will generate build configs
+ - After that Run command "kompose convert --provider=openshift --build=deploy-config --build-repo=https://github.com/bcgov/jag-traffic-courts-online.git" this will convert all the service inside the docker-compose.yaml to kubernetes yaml format and will generate deployment configs
+ 
```

Comment:
- Can we include the link to Kompose tool and how to install it
- Where will the yaml files be generated ? Can we include the path to look out for ?
-